lua/nvim-treesitter/parsers.lua

```diff
@@ -1179,6 +1179,16 @@ list.prisma = {
   maintainers = { "@elianiva" },
 }
 
+list.promql = {
+  install_info = {
+    url = "https://github.com/MichaHoffmann/tree-sitter-promql",
+    files = { "src/parser.c" },
+    branch = "main",
```

Comment:
if it's the default branch this can be removed

Reply:
...but you need to add the parser to lockfile.json

Reply:
:heavy_check_mark: